front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -1021,6 +1018,8 @@ export default function AppChat({
     void (async () => {
       const t = await updateTitle(title, m);
       await upsertChatSession(t);
+      void mutateChatSession();
```

Comment:
This is not directly related: this is just to make sure the list on the sidebar is updated when we create a new Chat, I noticed it was not the case 🙇🏻‍♀️